packages/css/src/components/mega-menu/mega-menu.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-mega-menu {
+  padding-block: var(--amsterdam-mega-menu-padding-block);
```

Comment:
Do we need this padding? Can't we do all padding with the Grids `paddingVertical` prop?

Reply:
That property only allows small, medium and large, and they are not the same as the padding for the menu.